src/Table/SynapseTableReflection.php

```diff
@@ -13,6 +13,7 @@
 use Keboola\TableBackendUtils\ReflectionException;
 use Keboola\TableBackendUtils\Table\Synapse\TableDistributionDefinition;
 use Keboola\TableBackendUtils\Table\Synapse\TableIndexDefinition;
+use Keboola\TableBackendUtils\TableNotExistsReflectionException;
 use function Keboola\Utils\returnBytes;
 
 final class SynapseTableReflection implements TableReflectionInterface
```

Comment:
Plus vzhledem k tomu, že je to nějaký obecný interface nad různýma databázema tak bych čekal, že taky bude nějaký obecný test toho interfacu, který se bude spouštět nad všema těma backendama... Má to nějaký důvod @zajca? 

Důvod proč je to důležité je, že zatím to takhle je implementované jen v synapsu a my budeme čekat, že se to stejně bude chovat i jinde a probublá tam nějaká podobná exception

Reply:
Tady jsme se domluvili, že ne, že to takhle nefunguje. Je to spíš takové "odkladiště metod" než že by to byl přímo adapter